lib/portlayer/storage/volume.go

```diff
@@ -70,10 +71,12 @@ type Volume struct {
 
 	// Backing device
 	Device Disk
+
+	CopyMode executor.CopyMode
 }
 
 // NewVolume creates a Volume
-func NewVolume(store *url.URL, ID string, info map[string][]byte, device Disk) (*Volume, error) {
+func NewVolume(store *url.URL, ID string, info map[string][]byte, device Disk, copyMode executor.CopyMode) (*Volume, error) {
```

Comment:
This probably doesn't need a backwards compatibility check given CopyNew is the default (by virtual of `0` being the nil value of `int`) but worth confirming that the backwards compatibility tests have a volume configuration that will exercise the copy paths in the future.